yarn.lock

```diff
@@ -11097,6 +11097,11 @@ typescript-estree@2.1.0:
     lodash.unescape "4.0.1"
     semver "5.5.0"
 
+typescript@^3.6.2:
```

Comment:
Can the `@typescript@~3.2.1` (line 11105) be removed?

Reply:
Removed